maxim/tests/test_add_dataset_entries.py

```
@@ -0,0 +1,348 @@
+                            "payload": "your content here",
+                        },
+                    },
+                ],
+            )
+
+            # Verify the network call was made
+            mock_client.request.assert_called_once()
+            
+            # Verify the request body contains properly converted entries
+            call_args = mock_client.request.call_args
+            self.assertIsNotNone(call_args)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)